app/views/content/non-uk-teachers/promos/_find-your-course.html.erb

```diff
@@ -6,10 +6,10 @@
   </div>
 
   <div class="right">
-    <span class="caption">Find your course</span>
+    <span class="caption-l caption--bold caption--purple">Find your course</span>
     <p>
-      You can search for postgraduate courses that lead to QTS (not including
-      Teach First).
+      You can search for postgraduate courses that lead to qualified teacher status (QTS), or find
```

Comment:
Not sure this entirely makes sense now re Teach First courses, particularly as it doesn't have a link. We really want people going through to Find? Do we need to mention Teach First here - is it confusing? We have a link to their site further up the page.

Happy for you to disagree though @EllieNodder !